src/libs/OptionsListUtils.js

```diff
@@ -650,7 +651,9 @@ function getOptions(
     const searchValue = parsedPhoneNumber.possible ? parsedPhoneNumber.number.e164 : searchInputValue.toLowerCase();
 
     // Filter out all the reports that shouldn't be displayed
-    const filteredReports = _.filter(reports, (report) => ReportUtils.shouldReportBeInOptionList(report, Navigation.getTopmostReportId(), false, iouReports, betas, policies));
+    const filteredReports = _.filter(reports, (report) =>
+        ReportUtils.shouldReportBeInOptionList(report, Navigation.getTopmostReportId(), false, iouReports, betas, policies, isShareDestination),
+    );
```

Comment:
It looks like filtering out admin-only reports is already happening on `main` - added in https://github.com/Expensify/App/pull/22157 - so we can remove this logic. Sorry for the duplicate work, I wasn't aware of that bug report until now

Reply:
Well, I will double check it. Thank you!

Reply:
Yes, the app already filters it. I've removed my logic for this.